<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create 2D tiff of cells</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonCellFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input cell file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCellFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>831</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>371</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonImageFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiffFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>831</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_voxel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_voxel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel Size (um)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_celltype">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_type1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell type 1</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_celltype</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_type2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell type 2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_celltype</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_typeboth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Both</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_celltype</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_zfraction">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_zthickness">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_zfraction">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Centre offset fraction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zthickness">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice thickness (um)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonImageFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tiffFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_voxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCellFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cellFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cell_tiffer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_zfraction</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_zthickness</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tiffFileSelecter()</slot>
  <slot>dataChanged()</slot>
  <slot>cell_tiffer()</slot>
  <slot>cellFileSelecter()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup_celltype"/>
 </buttongroups>
</ui>
